components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/DataFlowProcessingEngine.cs

```diff
@@ -175,28 +175,28 @@ public class DataFlowProcessingEngine : IProcessingEngine, IDisposable {
             _logger.Debug("Identity {deviceId}; {moduleId}", _identity.DeviceId, _identity.ModuleId);
 
             var diagInfo = new StringBuilder();
-            diagInfo.Append("\n   DIAGNOSTICS INFORMATION for         : {host}\n");
```

Comment:
Could we use AppendLine, rather than Append with \n`?

Reply:
Done